crates/core_arch/src/arm/neon/mod.rs

```diff
@@ -63,7 +63,7 @@ types! {
     /// ARM-specific 128-bit wide vector of four packed `u32`.
     pub struct uint32x4_t(u32, u32, u32, u32);
     /// ARM-specific 128-bit wide vector of four packed `f32`.
-    pub struct float32x4_t(f32, f32, f32, f32);
+    pub struct float32x4_t(pub f32, pub f32, pub f32, pub f32);
```

Comment:
Why did you change these fields to `pub`?

Reply:
Originally I was constructing the float32x4_t directly and needed these to be `pub` to do that. I forgot to push the version that had the `pub` removed when I switched to transmuting `f32x4`.